composer.json

```diff
@@ -14,9 +14,15 @@
 			"phpcbf -p"
 		]
 	},
+    "require": {
+		"automattic/jetpack-connection": "^1.51.9",
```

Comment:
Connection and Autoloader packages got new versions released recently, we could update them here.